packages/runtime-common/realm.ts

```diff
@@ -43,7 +43,8 @@ import {
 } from './stream';
 import { preprocessEmbeddedTemplates } from '@cardstack/ember-template-imports/lib/preprocess-embedded-templates';
 import * as babel from '@babel/core';
-import makeEmberTemplatePlugin from 'babel-plugin-ember-template-compilation';
+//@ts-ignore types are not found by consuming packages
```

Comment:
I tried everything I could to get types importing properly in `host` and `realm-server` but [these lint errors remained](https://github.com/cardstack/boxel/actions/runs/5281321329/jobs/9554729900), I’d love to not need this!